<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="mainTable">
        <property name="autoScrollMargin">
         <number>1</number>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="addScript">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="IconOK.qrc">
              <normaloff>:/OK/Icon/Add.svg</normaloff>:/OK/Icon/Add.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editScript">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="IconOK.qrc">
              <normaloff>:/OK/Icon/Edit.svg</normaloff>:/OK/Icon/Edit.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="deleteScript">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="IconOK.qrc">
            <normaloff>:/OK/Icon/Delete.svg</normaloff>:/OK/Icon/Delete.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>База данных</string>
    </property>
    <addaction name="choseDatabase"/>
    <addaction name="createNewDatabase"/>
    <addaction name="checkTables"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="choseDatabase">
   <property name="text">
    <string>Выбрать</string>
   </property>
  </action>
  <action name="createNewDatabase">
   <property name="text">
    <string>Создать новую</string>
   </property>
  </action>
  <action name="checkTables">
   <property name="text">
    <string>Проверить таблицы</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="IconOK.qrc"/>
 </resources>
 <connections/>
</ui>
